prime-dotnet-webapi/Services/OrganizationAgreementService.cs

```diff
@@ -89,6 +89,8 @@ public async Task<string> RenderOrgAgreementHtmlAsync(AgreementType type, string
                 (AgreementType.CommunityPharmacyOrgAgreement, true) => RazorTemplates.OrgAgreements.CommunityPharmacyPdf,
                 (AgreementType.CommunityPracticeOrgAgreement, false) => RazorTemplates.OrgAgreements.CommunityPractice,
                 (AgreementType.CommunityPracticeOrgAgreement, true) => RazorTemplates.OrgAgreements.CommunityPracticePdf,
+                (AgreementType.DeviceProviderOrgAgreement, false) => RazorTemplates.OrgAgreements.DeviceProvider,
+                (AgreementType.DeviceProviderOrgAgreement, true) => RazorTemplates.OrgAgreements.DeviceProviderPdf,
```

Comment:
When submitting the site I get these errors in the console, and the site submission is rejected. I am not sure if its related to the code above so it may be a red herring
<img width="939" alt="Capture" src="https://user-images.githubusercontent.com/45132887/127070308-b4eb8481-11c1-4617-ba79-ed41112cca1c.PNG">
<img width="522" alt="Capture2" src="https://user-images.githubusercontent.com/45132887/127070312-175bc06d-6b4c-44fb-a2e3-2cc10bb30361.PNG">

Reply:
So this may be you not having the most recent changes in the local environment. As I can't re-produce these locally.